MoltenVK/MoltenVK/GPUObjects/MVKImage.h

```diff
@@ -418,17 +525,20 @@ class MVKImageView : public MVKVulkanAPIDeviceObject {
 #pragma mark Metal
 
 	/** Returns the Metal texture underlying this image view. */
-	id<MTLTexture> getMTLTexture();
+	id<MTLTexture> getMTLTexture(uint8_t planeIndex) { return _planes[planeIndex]->getMTLTexture(); }
 
 	/** Returns the Metal pixel format of this image view. */
-	inline MTLPixelFormat getMTLPixelFormat() { return _mtlPixelFormat; }
+	inline MTLPixelFormat getMTLPixelFormat(uint8_t planeIndex) { return _planes[planeIndex]->_mtlPixFmt; }
+    
+    /** Returns the packed component swizzle of this image view. */
+    inline uint32_t getPackedSwizzle(uint8_t planeIndex) { return _planes[planeIndex]->_packedSwizzle; }
```

Comment:
A multi-plane view isn't going to have a swizzle, because you have to use a sampler Y'CbCr conversion with it, which has its own swizzle in it. You shouldn't need the `planeIndex` parameter here.